helpers/github.ts

```diff
@@ -257,3 +257,71 @@ export async function getProjectUrls() {
 
   return projectUrls;
 }
+
+// Function to calculate total rewards from open issues
+export const calculateTotalRewards = async (issues: GitHubIssue[]) => {
+  let totalRewards = 0;
+  await issues.forEach((issue) => {
+    const labels = issue.labels as {
+      id?: number | undefined;
+      node_id?: string | undefined;
+      url?: string | undefined;
+      name?: string | undefined;
+      description?: string | null | undefined;
+      color?: string | null | undefined;
+      default?: boolean | undefined;
+    }[];
```

Comment:
Can't you use a defined type from Octokit instead of manually defining it here?

Besides its redundant to write, for example, `default?: boolean | undefined;`

You should do something like `default: boolean` and ensure `true` or `false` values for type safety.

Reply:
the label type was not exported from octokit, so I just copied the object

Reply:
I think a custom label was already declared for this, i'd use that instead